Failed tests (12513): TestNodeStatus

The following test appears to have failed:

[#12513](https://circleci.com/gh/cockroachdb/cockroach/12513):

```
[Trace sending batch]
 *  00:20:57.024678 meta descriptor lookup
 *  00:20:57.024713 sending RPC
 *  00:20:57.024801 sending to 127.0.0.1:34069
I0209 00:20:57.043209 125 storage/engine/rocksdb.go:137  closing in-memory rocksdb instance
--- FAIL: TestNodeStatus (0.77s)
	cli_test.go:822: value for live_bytes (10457) greater than max (10000)
	cli_test.go:822: value for live_bytes (10457) greater than max (10000)
=== RUN   TestGenMan
Generated CockroachDB man pages in /tmp/TestGenMan656001801/
--- PASS: TestGenMan (0.17s)
	<autogenerated>:28: prior leak detected, leaktest disabled
=== RUN   TestGenAutocomplete
Generated bash completion file /tmp/TestGenAutoComplete235715796/cockroach.bash
--- PASS: TestGenAutocomplete (0.06s)
	<autogenerated>:28: prior leak detected, leaktest disabled
--
 *  00:21:12.786117 meta descriptor lookup
 *  00:21:12.786161 sending RPC
 *  00:21:12.786259 sending to 127.0.0.1:43949
Error: Error: node 10000 doesn't exist
--- PASS: Example_node (0.81s)
FAIL
FAIL	github.com/cockroachdb/cockroach/cli	17.601s
=== RUN   TestClientTxnSequenceNumber
--- PASS: TestClientTxnSequenceNumber (0.00s)
=== RUN   TestTxnRequestTxnTimestamp
--- PASS: TestTxnRequestTxnTimestamp (0.01s)
=== RUN   TestTxnResetTxnOnAbort
--- PASS: TestTxnResetTxnOnAbort (0.00s)
=== RUN   TestTransactionConfig
--- PASS: TestTransactionConfig (0.00s)
=== RUN   TestCommitReadOnlyTransaction
--- PASS: TestCommitReadOnlyTransaction (0.01s)

```
Please assign, take a look and update the issue accordingly.

--- a/cli/cli_test.go
+++ b/cli/cli_test.go
@@ -799,11 +799,11 @@ func checkNodeStatus(t *testing.T, c cliTest, output string, start time.Time) {
 		idx    int
 		maxval int64
 	}{
-		{"live_bytes", 4, 10000},
-		{"key_bytes", 5, 10000},
-		{"value_bytes", 6, 10000},
-		{"intent_bytes", 7, 10000},
-		{"system_bytes", 8, 10000},
+		{"live_bytes", 4, 20000},
+		{"key_bytes", 5, 20000},
+		{"value_bytes", 6, 20000},
+		{"intent_bytes", 7, 20000},
+		{"system_bytes", 8, 20000},
 		{"leader_ranges", 9, 3},
 		{"repl_ranges", 10, 3},
 		{"avail_ranges", 11, 3},

